transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,135 @@
+import requests
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def britishize_americanize(string, final_dict):
+    """
+    Parameters:
+        string(str): original string
+        final_dict(dict): dictionary with all the different possible words in american and british english
+    Returns:
+        str: String after replacing the words
+    """
+    string = " ".join([final_dict.get(word, word) for word in string.split()])
+    return string
+
+
+class AmericanizeBritishizeEnglish(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION, TaskType.SENTIMENT_ANALYSIS,
+             TaskType.QUESTION_ANSWERING, TaskType.QUESTION_GENERATION, TaskType.PARAPHRASE_DETECTION]
+    languages = ["en"]
```

Comment:
Please add the revelant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) for your PR too.

Reply:
Done. Kindly check.

Reply:
Hi @kaustubhdhole ,
Kindly let me know if any other changes are required. Thanks.

Reply:
Overall, I think this seems fine to merge.. just add the Data and code section.